examples/viewer.cpp

```diff
@@ -135,7 +150,7 @@ bool Viewer::render()
     // wipe the drawing surface clear
     glClear(GL_COLOR_BUFFER_BIT | GL_DEPTH_BUFFER_BIT);
 
-    GLint x = 0, y = 0, width = 600, height = 400;
```

Comment:
Since the width here for each viewport was 600, shouldn't the total window width be 1200, and not 1280?